src/pages/SubmissionDetails.test.js

```diff
@@ -222,7 +222,8 @@ describe('Submission Details page', () => {
 
 
     await screen.getByText(/Submission e8347247-4738-4ad1-a591-56c119f93f58/)
-    await screen.getByText(/workflow: Hello world/)
+    await screen.getByText(/Submission name: hello world/)
+    await screen.getByText(/Workflow name: Hello world/)
```

Comment:
Maybe we can change one of these names to prove that the UI elements are picking things out from the right parts of the API response?